.gitlab-ci-check-docker-acceptance.yml

```diff
@@ -135,11 +135,10 @@ publish:acceptance:
     #  According to https://docs.coveralls.io/supported-ci-services
     #  we should set CI_NAME, CI_BUILD_NUMBER, etc. But according
     #  to goveralls source code (https://github.com/mattn/goveralls)
-    #  many of these are not supported. Set CI_BRANCH, CI_PR_NUMBER,
+    #  many of these are not supported. Set CI_BRANCH
     #  and pass few others as command line arguments.
     #  See also https://docs.coveralls.io/api-reference
     - export CI_BRANCH=${CI_COMMIT_BRANCH}
-    - export CI_PR_NUMBER=${CI_COMMIT_BRANCH#pr_}
```

Comment:
FYI; this env variable is not required according to the API specification, and also breaks the API if it is not a `pr_*` branch (which is why I think the coverage is not reported).

Reply:
Okey, thanks. Maybe something has changed in either `goveralls` or the Coveralls API during this year, because I remember this being necessary to be exported when I originally made the migration.